ports/esp32/Makefile

```diff
@@ -466,6 +494,7 @@ OBJ = $(OBJ_MP)
 
 APP_LD_ARGS =
 APP_LD_ARGS += $(LDFLAGS_MOD)
+APP_LD_ARGS += $(addprefix -T,$(LD_FILES))
```

Comment:
What's `LD_FILES` intended to be used for?  The esp32 is very specific about the linking phase, although I guess the user could add some additional bits if they really need to.

Reply:
@nevercast asked for it. My guess is that it might be related to the rtc mem stuff.